mockgcp/mockiampolicies.go

```diff
@@ -75,17 +99,23 @@ func (m *mockIAMPolicies) serveSetIAMPolicy(resourcePath string, httpRequest *ht
 		return &http.Response{StatusCode: http.StatusConflict, Body: responseBody}, nil
 	}
 
-	request.Policy.Version = 3
+	// conditional role bindings must specify version 3
+	hasConditions := false
+	for _, binding := range request.Policy.Bindings {
+		if binding.Condition != nil {
+			hasConditions = true
```

Comment:
might as well break at this point. No point in going through the rest of the bindings....

Reply:
Done!